### PR TITLE
Add an initial test for wasm multi-value.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Comment by Ms2ger** [comment]
The last test appears to work in the V8 shell when passing `--experimental-wasm-mv`; I haven't found a way to enable the code within Chromium. The integration with the JS API is not yet implemented anywhere, AFAICT.

CC @rossberg

File: wasm/jsapi/constructor/multi-value.any.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    .exportFunc();
+
+  const buffer = builder.toBuffer();
+
+  const result = await WebAssembly.instantiate(buffer);
+  const swapped = result.instance.exports.swap(7, 4.2);
+  assert_true(Array.isArray(swapped));
+  assert_equals(Object.getPrototypeOf(swapped), Array.prototype);
+  assert_array_equals(swapped, [4.2, 7]);
+}, "multiple return values from js to wasm");
```
Comment (littledan) [inline wasm/jsapi/constructor/multi-value.any.js]: I don't understand how this is JS to Wasm. Did you mean to make further changes to this test?

**Review by littledan — COMMENTED** [review]
Tests look correct to me, but maybe you meant to include further coverage. To pass V8 flags from Chrome invoked on the command line, use `--js-flags="--experimental-wasm-mv"`.

File: wasm/jsapi/constructor/multi-value.any.js (hunk 1 above)
Comment (Ms2ger) [reply wasm/jsapi/constructor/multi-value.any.js]: It isn't, indeed. I removed this part; I'll submit a JS to wasm test at some later point.

File: wasm/jsapi/constructor/multi-value.any.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+// META: global=jsshell
+// META: script=/wasm/jsapi/wasm-module-builder.js
+// META: script=/wasm/jsapi/assertions.js
+
+promise_test(async () => {
+  const type_if_fi = makeSig([kWasmF32, kWasmI32], [kWasmI32, kWasmF32]);
```
Comment (xtuc) [inline wasm/jsapi/constructor/multi-value.any.js]:
```suggestion
  const type_if_fi = makeSig([kWasmI32, kWasmF32], [kWasmF32, kWasmI32]);
```

Confims with name and usage line 22: `result.instance.exports.swap(7, 4.2);`

Comment (xtuc) [reply wasm/jsapi/constructor/multi-value.any.js]: Also type_if_fi is incorrectly defined multiple times in the file, could we hoist it?

Comment (Ms2ger) [reply wasm/jsapi/constructor/multi-value.any.js]: Good catch! Swapped the arguments instead, and moved to f64 to avoid rounding errors.

**Review by xtuc — APPROVED** [review]

**Pull request merged**